pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.github.danielflower.mavenplugins</groupId>
   <artifactId>gitlog-maven-plugin</artifactId>
-  <version>1.13-SNAPSHOT</version>
+	<version>1.13.4-SNAPSHOT</version>
```

Comment:
Remove the `.4` - this uses the non-standard maven release plugin that generates revision numbers based on git tags.